pandas/tests/window/test_numba.py

```diff
@@ -121,3 +121,71 @@ def test_invalid_kwargs_nopython():
         Series(range(1)).rolling(1).apply(
             lambda x: x, kwargs={"a": 1}, engine="numba", raw=True
         )
+
+
+@td.skip_if_no("numba", "0.46.0")
```

Comment:
maybe put in a new file test_table.py?

Reply:
I was hoping to keep all numba related tests together. 

Maybe once table is available for non numba engine I can move to a new file